README.md

```diff
@@ -1,8 +1,12 @@
 [![Build Status](https://ci.vmware.run/api/badges/vmware/docker-volume-vsphere/status.svg)](https://ci.vmware.run/vmware/docker-volume-vsphere) [![Join the chat at https://gitter.im/vmware/docker-volume-vsphere](https://badges.gitter.im/vmware/docker-volume-vsphere.svg)](https://gitter.im/vmware/docker-volume-vsphere?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
+[<img src="https://github.com/vmware/docker-volume-vsphere/blob/docUpdateManagedPlugin.shuklanirdesh82/docs/misc/Docker%20Certified.png" width="96">](https://store.docker.com/plugins/vsphere-docker-volume-service?tab=description)
```

Comment:
Can it be added as badge?

Link is pointing to private branch. It should point to master.